.github/workflows/release-runtime.yml

```diff
@@ -341,17 +193,18 @@ jobs:
           name: shibuya-runtime
           path: runtime-artifacts
 
-      - name: Use Node.js 18.x
+      - name: Use Node.js 20.x
         uses: actions/setup-node@v2
         with:
-          node-version: 18.x
+          node-version: 20.x
 
-      - name: Get the latest release
+      - name: Get the latest runtime release
         id: latest-release
-        uses: pozetroninc/github-action-get-latest-release@v0.5.0
-        with:
-          repository: AstarNetwork/Astar
-          excludes: "prerelease, draft"
+        # We're making an assumption that the latest runtime release will be within the last 30 releases
+        run: |
+          latest_runtime_tag=$(curl -s https://api.github.com/repos/AstarNetwork/Astar/releases | \
```

Comment:
Quick question: Given this reverse chronological releases:

- v3.2.1
- v3.2.0
- runtime-1000
- v3.1.0

This command will return: `latest_runtime_tag=v3.2.1`. But we actually want `runtime-1000`, right?

Reply:
The regex should first try and match the `runtime` tag, only then will it look for `vXYZ` tag.

So for your example, it will match `runtime-1000`.

Reply:
I agree on the desired output, but are you sure of the implementation? I am not a regex expert but with this pipe annotation, `v3.2.1` tag comes first using my previous example.

What about looking first for `runtime-XXXX` tags only, something like this:

```bash
runtime_tags=$(echo "$releases" | jq -r 'map(select(.name | test("^runtime-\\d{4,}$"; "i")))[0] | .tag_name')

semantic_tags=$(echo "$releases" | jq -r 'map(select(.name | test("^v\\d+\\.\\d+\\.\\d+$"; "i")))[0] | .tag_name')
    
if [ -z "$runtime_tags" ]; then
  latest_runtime_tag=$(echo "$semantic_tags" | head -n 1)
else
  latest_runtime_tag=$(echo "$runtime_tags" | head -n 1)
fi
```

Reply:
You're right, the way the test/search works for `jq` is not as I assumed.

I will apply your suggestion - later on we can clean it up since it will only be relevant for the first release.
Good catch :)